src/main/org/codehaus/groovy/tools/javac/JavaStubGenerator.java

```diff
@@ -925,7 +931,8 @@ private void printImports(PrintWriter out, ClassNode classNode) {
         imports.addAll(Arrays.asList(ResolveVisitor.DEFAULT_IMPORTS));
 
         for (Map.Entry<String, ImportNode> entry : moduleNode.getStaticImports().entrySet()) {
-            imports.add("static "+entry.getValue().getType().getName()+"."+entry.getKey());
+            if (entry.getKey().equals(entry.getValue().getFieldName()))
```

Comment:
I'm using a strategy similar to the one used for regular imports - excluding aliased static imports, and replacing their occurrences with their fully qualified names.